### ISSUE TITLE
[3.0.0-rc0] Error in template file separators on Windows

### ISSUE THREAD

**Issue by NanoWar** [swagger-api/swagger-codegen#7832]
Hey, I just tried the new 3.0.0-rc0 version with my spec, but could not create anything, because - I assume - the template files cannot be found, see exception below. I'm on Windows, which has different file separators. Could that be the issue?

running:
`java -jar "swagger-codegen-cli-3.0.0-rc0.jar" generate -i my.json -l inflector -o generated`

```
Exception in thread "Thread-1" java.lang.RuntimeException: Could not generate model 'MyModel'
        at io.swagger.codegen.DefaultGenerator.generateModels(DefaultGenerator.java:407)
        at io.swagger.codegen.DefaultGenerator.generate(DefaultGenerator.java:722)
        at io.swagger.codegen.cmd.Generate.run(Generate.java:312)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.io.FileNotFoundException: /v2/JavaInflector/\model.mustache
        at com.github.jknack.handlebars.io.URLTemplateLoader.sourceAt(URLTemplateLoader.java:70)
        at com.github.jknack.handlebars.Handlebars.compile(Handlebars.java:357)
        at com.github.jknack.handlebars.Handlebars.compile(Handlebars.java:343)
        at io.swagger.codegen.DefaultGenerator.getHandlebars(DefaultGenerator.java:1013)
        at io.swagger.codegen.DefaultGenerator.processTemplateToFile(DefaultGenerator.java:738)
        at io.swagger.codegen.DefaultGenerator.generateModels(DefaultGenerator.java:394)
        ... 3 more
```

`/v2/JavaInflector/\model.mustache` looks suspicious

**Comment by HugoMario** [comment]
@NanoWar , this issue was fixed on latest commits for `3.0.0-SNAPSHOT`.

**Comment by NanoWar** [comment]
Thank you! And keep up the great work :)
